index.yaml

```diff
@@ -1229,21 +1268,21 @@ entries:
       catalog.cattle.io/namespace: cattle-gatekeeper-system
       catalog.cattle.io/release-name: rancher-gatekeeper-crd
     apiVersion: v1
-    created: "2021-06-11T16:24:25.451369-07:00"
+    created: "2021-06-18T18:28:25.870544-07:00"
     description: Installs the CRDs for rancher-gatekeeper.
-    digest: 65361c16b473e1ecf0ad889e507bb52daacf2e022ad95bd2012d5ff48b3ce1e2
+    digest: d82c0a0eae6ef19cd815fa0f78730403fac042c886c4564af1a0935d9be54d08
     name: rancher-gatekeeper-crd
     type: application
     urls:
-    - assets/rancher-gatekeeper/rancher-gatekeeper-crd-3.3.1+up3.3.0.tgz
-    version: 3.3.1+up3.3.0
+    - assets/rancher-gatekeeper/rancher-gatekeeper-crd-3.3.001.tgz
+    version: 3.3.001
```

Comment:
3.3.001 is a released version https://github.com/rancher/charts/tree/release-v2.5/charts/rancher-gatekeeper/rancher-gatekeeper

so we are good here